### PR TITLE
fix: description of massdns

### PR DESCRIPTION
**Pull request by 0xflotus**
I don't know why, maybe a Copy&Paste-Error, but the description of `massdns` is potentially wrong.
Referring to the [official repository](https://github.com/blechschmidt/massdns) it is actually a `high-performance DNS stub resolver` instead of `Particle library for 3D graphics`

### CODE REVIEW

File: Formula/massdns.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Massdns < Formula
-  desc "Particle library for 3D graphics"
+  desc "A high-performance DNS stub resolver"
```
Comment (carlocab) [inline Formula/massdns.rb:2]:
```suggestion
  desc "High-performance DNS stub resolver"
```

**Review by carlocab — COMMENTED** [review]
Particle library, DNS stub resolver, what's the difference? 🙃

**Comment by carlocab** [comment]
Thanks for your contribution, @0xflotus!

**Comment by chenrui333** [comment]
yeah, copy paste error 😂  thanks for the fix.

**Pull request merged**